README.md

```diff
@@ -159,18 +159,19 @@ in URL-encoded or base64 format, which can be indicated by prefixing the data as
 * if not prefixed, then assume raw (no encoding)
 
 ### Enable @SQLCHECKBOX action tag
-Allow use of the **@SQLCHECKBOX** action tag for checkbox fields, which will dynamically replace
-the checkbox options with those from a specified SQL field. The format must follow the pattern
-@SQLCHECKBOX='????', in which the desired value must be the field name of an SQL field in the
-project.<br>Note: Checkbox options will **not** be replaced if the form, record or project has been
-locked.
+Allow use of the **@SQLCHECKBOX** action tag for SQL fields, which will replace the drop-down list
+with a checkbox for each option similar to a checkbox field.
 
-*Note that using this action tag will cause the project data dictionary to be updated as necessary
-in order to align the checkbox options with those from the SQL field.*
+When writing an SQL query for use with the *@SQLCHECKBOX* action tag, ensure that the returned
+column names are `code` and `label`, otherwise the query will not function. Also note that the
+*@SQLCHECKBOX* action tag does not work with the *@IF* action tag.
```

Comment:
As I found out the other day, this doesn't work with a semi-colon on the end of the sql statement.  MySQL is so fussy about it's semi-colons I'm used to always using them.  I think it would help for the README/Action Tag guide to include a warning NOT to include final semi-colon, as it doesn't then stop with a meaningful error, it just doesn't return any rows.

Reply:
I have amended the SQL transformation for the `@SQLCHECKBOX` action tag so any trailing semi-colons are now automatically stripped off.